src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -189,8 +189,19 @@ public CurlHandler()
             _agent = s_singletonSharedAgent ?? new MultiAgent(this);
         }
 
-        /// <summary>Overridden by another partial implementation to set <see cref="result"/> to true if a single MultiAgent should be used.</summary>
-        static partial void UseSingletonMultiAgent(ref bool result);
+        /// <summary>Set <see cref="result"/> to true if a single MultiAgent should be used.</summary>
+        static void UseSingletonMultiAgent(ref bool result)
```

Comment:
The method signature previously used `ref bool result` to take advantage of partial methods, which can return a result.  Now that it's no longer partial, this can be changed to:
```C#
private static bool UseSingletonMultiAgent()
```
and the call site can be changed from:
```C#
bool useSingleton = false;
UseSingletonMultiAgent(ref useSingleton);
if (useSingleton)
```
to:
```C#
if (UseSingletonMultiAgent())
```
It could also be made into a property, or even just moved to the call site rather than having a method for it.

My main point is, this dance with the ref argument should no longer be necessary.